readme.md

```diff
@@ -146,6 +146,7 @@
 	- [Scientific Audio](https://github.com/faroit/awesome-python-scientific-audio#readme) - Scientific research in audio/music.
 	- [CircuitPython](https://github.com/adafruit/awesome-circuitpython#readme) - A version of Python for microcontrollers.
 	- [Data Science](https://github.com/krzjoa/awesome-python-data-science#readme) - Data analysis and machine learning.
+	- [Typing](https://github.com/typeddjango/awesome-python-typing) - Types and tools to work with them for Python.
```

Comment:
```suggestion
	- [Typing](https://github.com/typeddjango/awesome-python-typing#readme) - Types and tools to work with them for Python.
```

And the description should be about what "typing" means for Python, not what the list contains.

Reply:
Ok, changed the wording. Thanks!